### ISSUE TITLE
Revive output_parallel_statistics and make it a plugin

### ISSUE THREAD

**Issue by tjhei** [geodynamics/aspect#1052]
see
```
$ grep output_para -i -r  source/
source/global.cc:  const bool output_parallel_statistics = false;
source/simulator/helper_functions.cc:    if (!aspect::output_parallel_statistics)
```

Turn this into a postprocessor that can be turned on instead of a variable that you have to modify by hand. Make sure it works, maybe extend it.

**Comment by gassmoeller** [comment]
@spco is this issue closed by #1062 or is there something else coming?
Thanks for working on this by the way, I will definitely use this postprocessor for some of my runs :+1:

**Comment by spco** [comment]
This can be closed - the matrix statistics I'm working on will go towards #1053.
